imod_coupler/drivers/dfm_metamod/dfm_metamod.py

```diff
@@ -289,14 +291,24 @@ def update(self) -> None:
         # flux from modflow to dflow 1d
         self.exchange_V_1D()
 
+        q_corr_init = self.dfm.get_cumulative_fluxes_1d_nodes()
+        assert q_corr_init is not None
+
         # sub timestepping between metaswap and dflow
-        subtimestep_endtime = self.get_current_time()
+        subtimestep_endtime = tBegin
```

Comment:
```suggestion
        subtimestep_endtime = t_begin
```

Same for other occurrences of `tBegin`

Reply:
done